container_pipeline/workers/linter.py

```diff
@@ -81,15 +81,37 @@ def lint(self):
             out, err = run_cmd_out_err(command)
             if err == "":
                 response = self.handle_lint_success(out)
+                self.logger.info(
+                    "Docker file linting successful going for job creation")
+                if not response["job_created"]:
+                    self.logger.warning(
+                        "Openshift project is not created putting it back to tube")
+                    if self.job.get("retry") is None:
+                        self.job["retry"] = 1
+                    else:
+                        self.job["retry"] = self.job.get("retry") + 1
+                    self.job["action"] = "start_linter"
+                else:
+                    self.logger.info("Deleting 'dockerfile' data from job")
+                    self.job["dockerfile"] = None
+                    self.job["retry"] = None
             else:
                 response = self.handle_lint_failure(err)
                 self.job["dockerfile"] = None
                 self.job["action"] = "notify_user"
-                self.queue.put(json.dumps(self.job), 'master_tube')
+
+            if self.job.get("retry") > self.MAX_RETRY:
```

Comment:
Comparing `None` with integer when we're able to successfully create an OpenShift project in first attempt is going to break things here and we're going to see exception in the logs. Also breaks the flow for particular job, IMHO.

Reply:
Okay, I take it back, it does work:

```python
In [1]: None > 1
Out[1]: False

In [2]: None > 10
Out[2]: False

In [3]: None < 10
Out[3]: True
```